packages/fluxible-addons-react/connectToStores.js

```diff
@@ -59,6 +59,7 @@ function createComponent(Component, stores, getStateFromStores, customContextTyp
     });
 
     hoistNonReactStatics(StoreConnector, Component);
+    StoreConnector.WrappedComponent = Component;
```

Comment:
Thanks for taking a look around at other libraries. I agree we should try to align with the rest of the community. Let's keep this PR the way it is.

For now, maybe we attach both `wrappedComponent` and `WrappedComponent` in the other files to maintain backwards compatibility but provide a way forwards.  Then we'll take a look at removing the lowercase version in our other HoCs in the next breaking versions. 

Not ideal, but at least better aligned.